subgraph/src/public-lock.ts

```diff
@@ -399,3 +406,16 @@ export function handleLockMetadata(event: LockMetadataEvent): void {
     lock.save()
   }
 }
+
+export function handleReferrerFees(event: ReferrerFeeEvent): void {
```

Comment:
This function tracks all referrer fee events for your "lock". If the "lock" exists, it logs the referrer fee set for an address. This way, we can monitor the "fee", "referrer", and associated "lock".